plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -926,34 +931,37 @@ func TestLimitRanger_GetLimitRangesFixed22422(t *testing.T) {
 			}
 		}()
 	}
+
+	// let more goroutines enter Do
+	wg1.Wait()
+
+	// it can make the test more stable under a large number of stress tests
+	// the absence of this can lead to occasional lock gap problems under extensive stress testing
+	time.Sleep(time.Millisecond * 10)
```

Comment:
This is a smell that the async aspects of the test aren't fully correct. Can we write the test in a way such that this isn't needed?

Reply:
If the test is flaking badly, I'd rather skip it for now while this is worked on than add a 10 ms sleep

Reply:
https://github.com/golang/sync/blob/master/singleflight/singleflight_test.go#L62

A similar test exists in the golang code

Reply:
The current test situation is still a relatively low problem, generally speaking, the test has problems, retesting can solve the problem

Reply:
retesting is a problem, we want to keep a high bar and avoid flaky tests

Reply:
I'm verifying a new method and am testing it with stress, if there are no errors in the long case, I will commit